damus/Views/ConfigView.swift

```diff
@@ -42,6 +42,23 @@ struct ConfigView: View {
         ZStack(alignment: .leading) {
             Form {
                 if let ev = state.contacts.event {
+                    Section {
+                        if let relays = decode_json_relays(ev.content) {
+                            List(Array(relays.keys.sorted()), id: \.self) { relay in
+                                RelayView(state: state, ev: ev, relay: relay)
+                            }
+                        }
+                    } header: {
+                        HStack {
+                            Text("Relays")
+                            Spacer()
+                            Button(action: { show_add_relay = true }) {
+                                Image(systemName: "plus")
+                                    .foregroundColor(.accentColor)
+                            }
+                        }
+                    }
+
                     Section("Relays") {
```

Comment:
Do we need to remove this section now since you have already included it above?
I see in your video two identical sets of relays.

Reply:
You're right, I must have forgot to push that commit. Thanks, should be good now.
![Simulator Screen Recording - iPhone 14 Pro - 2022-12-29 at 17 53 27](https://user-images.githubusercontent.com/264977/210026775-7165d60b-abf2-44de-a74a-b4c7f014e53d.gif)